Makefile

```diff
@@ -0,0 +1,40 @@
+SQLFLOW_VERSION := develop
+SHELL := /bin/bash
+
+setup:
+	python3 -m venv venv
+	source venv/bin/activate \
+	&& $(MAKE) install-requirements protoc
+
+install-requirements:
+	pip install -U -e .
+
+test:
+	python3 setup.py test
+
+clean:
+	rm -rf build dist .eggs *.egg-info .pytest_cache sqlflow/proto
+
+protoc:
+	python3 -m venv build/grpc
+	source build/grpc/bin/activate \
+	&& pip install grpcio-tools \
+	&& mkdir -p build/grpc/sqlflow/proto \
+	&& curl -o build/grpc/sqlflow/proto/sqlflow.proto \
+		https://raw.githubusercontent.com/wangkuiyi/sqlflowserver/$(SQLFLOW_VERSION)/sqlflow.proto \
+	&& python -m grpc_tools.protoc -Ibuild/grpc --python_out=. \
+		--grpc_python_out=. build/grpc/sqlflow/proto/sqlflow.proto \
+	&& rm -rf build/grpc
```

Comment:
Why `rm -rf build/grpc` in the end?

Reply:
Looks like `make protoc` will generate grpc client at `sqlflow/proto`

Reply:
Yes, and I like keep folded clean